lib/document/DocumentFacade.js

```diff
@@ -87,11 +89,14 @@ class DocumentFacade {
   /**
    * Create Documents State Transition
    *
-   * @param {Document[]} documents
+   * @param {string} contractId
+   * @param {string} ownerId
+   * @param {SubTransitionsMap} subTransitionsMap
+   *
    * @return {DocumentsStateTransition}
    */
-  createStateTransition(documents) {
-    return this.factory.createStateTransition(documents);
+  createStateTransition(contractId, ownerId, subTransitionsMap) {
```

Comment:
We passing models here so they have contract and owner inside. Don’t forget to check that all Models have the same Contract and owner